third_party/terraform/website/docs/r/logging_billing_account_bucket_config.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Stackdriver Logging"
+layout: "google"
+page_title: "Google: google_logging_billing_account_bucket_config"
+sidebar_current: "docs-google-logging-billing-account-bucket-config"
+description: |-
+  Manages a billing account level logging bucket config.
+---
+
+# google\_logging\_billing_account\_bucket\_config
+
+Manages a billing account level logging bucket config. For more information see
+[the official logging documentation](https://cloud.google.com/logging/docs/) and
+[Storing Logs](https://cloud.google.com/logging/docs/storage).
+
+Logging buckets are automatically created for a given folder, project, organization, billingAccount and cannot be deleted. Creating a resource of this type will acquire and update the resource that already exists at the desired location. These buckets cannot be removed so deleting this resource will remove the bucket config from your terraform state but will leave the logging bucket unchanged. The buckets that are currently automatically created are "_Default" and "_Required".
```

Comment:
Let's call this out with a -> (note) or ~> (important) like we do with the KMS resources, to make sure people don't gloss over it